app/layout.tsx

```diff
@@ -0,0 +1,27 @@
+import type { Metadata } from 'next';
+import './layout.css';
+import 'bootstrap/dist/css/bootstrap.css';
+import Header from '@/components/Header';
+import Footer from '@/components/Footer';
+
+export const metadata: Metadata = {
+  title: 'Support Local Akron - Show Your Love ❤️',
```

Comment:
Trying to think through the easiest way to make this a template for others. We'll probably want to pull this info into a larger content file. Maybe even in i18n file, but that's far over-optimizing I think for now.